.buildkite/pipelines/pull_request.yml

```diff
@@ -0,0 +1,17 @@
+env:
+  GITHUB_COMMIT_STATUS_ENABLED: 'true'
+  GITHUB_COMMIT_STATUS_CONTEXT: 'buildkite/kibana-pull-request'
+steps:
+  - command: .buildkite/scripts/lifecycle/pre_build.sh
+    label: Pre-Build
+
+  - wait
+
+  - command: echo 'Hello World'
```

Comment:
Still needed?

Reply:
Yeah, it's intended to be a simple, easy to run sanity-check pipeline until the pull request pipeline is built out.